plugins/zenoh-plugin-rest/src/lib.rs

```diff
@@ -419,7 +420,7 @@ async fn query(mut req: Request<(Arc<Session>, String)>) -> tide::Result<Respons
                 .content_type()
                 .map(|m| Encoding::from(m.to_string()))
                 .unwrap_or_default();
-            query = query.with_value(Value::from(body).with_encoding(encoding));
+            query = query.with_value(Value::from(body).encoding(encoding));
```

Comment:
`with_value` should be `value`

Reply:
Agree. I reworked `Value` API: added `value` method to `ValueBuilderTrait`. I.e. each object whcih accepts payload/encoding accepts value also. For `GetBuilder` added special treatment of emply value. When value is empty, setter behaves like value is set to `None`. This allows to write code like in current "z_get" example:
```Rust
    let (config, selector, value, target, timeout) = parse_args();
...
    let replies = session
        .get(&selector)
        .value(value)
        .target(target)
        .timeout(timeout)
        .res()
        .await
        .unwrap();
 ```
instead of 
```Rust
    let replies = match value {
        Some(value) => session.get(&selector).with_value(value),
        None => session.get(&selector),
    }
    .target(target)
    .timeout(timeout)
    .res()
    .await
    .unwrap();
```